core/storage/classifier/gae_models_test.py

```diff
@@ -14,156 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import datetime
-
 from core.domain import classifier_domain
 from core.platform import models
 from core.tests import test_utils
-import feconf
 
 (classifier_models,) = models.Registry.import_models(
     [models.NAMES.classifier])
 
 
-class ClassifierTrainingJobModelUnitTests(test_utils.GenericTestBase):
-    """Test the ClassifierTrainingJobModel class."""
-
-    def test_create_and_get_new_training_job_runs_successfully(self):
-        next_scheduled_check_time = datetime.datetime.utcnow()
-        job_id = classifier_models.ClassifierTrainingJobModel.create(
-            'LDAStringClassifier', 'TextInput', 'exp_id1', 1,
```

Comment:
You will have to change LDAStringClassifier to TextClassifier as you did in feconf.